[package]
name = "pyerror"
version = "0.1.0"
edition = "2021"

[dependencies]
auth = { path = "../../../../lib/auth" }
cpython_ext = { path = "../../../../lib/cpython-ext", default-features = false }
cpython = { version = "0.7", default-features = false }
dag = { path = "../../../../lib/dag" }
edenapi = { path = "../../../../lib/edenapi" }
hgcommits = { path = "../../../../lib/hgcommits" }
http-client = { path = "../../../../lib/http-client" }
indexedlog = { path = "../../../../lib/indexedlog" }
metalog = { path = "../../../../lib/metalog" }
repolock = { path = "../../../../lib/repolock" }
revisionstore = { path = "../../../../lib/revisionstore" }
revlogindex = { path = "../../../../lib/revlogindex" }
treestate = { path = "../../../../lib/treestate" }
types = { path = "../../../../lib/types" }
